goldenmaster/apigear/mqtt/mqttcppclient.h

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <queue>
+#include <set>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <string>
+#include <atomic>
+#include <thread>
+#include <condition_variable>
+#include "nlohmann/json.hpp"
+#include "mqttcommon.h"
+#include "mqtttopic.h"
+
+typedef void* MQTTAsync;
+
+namespace ApiGear {
+namespace MQTTImpl {
```

Comment:
True, we just used the PocoImpl namespace to differeniate between the Poco namespace and our implementation.
Does not matter here since there is no other MQTT namespace